dotnet/samples/GettingStartedWithProcesses/Step00/Steps/EndStep.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel;
+
+namespace Step00.Steps;
+
+public sealed class EndStep : KernelProcessStep
```

Comment:
nit - We have an internal `EndStep` that's more of an implementation detail.  Just to avoid any edge-confusion, I wonder if `LastStep` might be adequate here?